doc/examples/plot_multiblock_local_binary_pattern.py

```diff
@@ -0,0 +1,82 @@
+"""
+===========================================================
+Multi-Block Local Binary Pattern for texture classification
+===========================================================
+
+In this example, we will see how to compute the multi-block
+local binary pattern at a specified image and how to visualize it.
+
+The features are calculated in a way similar to local binary
```

Comment:
The features are calculated similarly to local binary patterns, except that summed blocks are used instead of individual pixel values.